test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -1038,10 +1038,10 @@ def addSuccess(self, test):
     def _cleanup(self):
         self.logger.info('Cleaning up temporary files')
         du = DshUtils()
-        hosts = self.param_dict['moms']
+        hosts = set(self.param_dict['moms'])
         for server in self.param_dict['servers']:
-            if server not in self.param_dict['moms']:
-                hosts.update(self.param_dict['servers'])
+            if server not in hosts:
+                hosts.extend(server)
```

Comment:
Python `set` doesn't have `.extend()`:
```
>>> t = set()
>>> t.extend('a')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'set' object has no attribute 'extend'
>>> 
```

So your newly modified changes are not getting tested in whatever test your are running. Means you need to run test with `-pmoms=<moms on remote host>,servers=<server on host other than mom>,nomom=<server hostname>` to `pbs_benchpress`